js/web/lib/wasm/jsep/webgpu/ops/pool.ts

```diff
@@ -307,21 +309,21 @@ const createAveragePoolProgramInfo =
       };
     };
 
-export const parseAveragePoolAttributes = (attributes: Record<string, unknown>): AveragePoolAttributes => {
+const parseAveragePoolAttributes = (attributes: Record<string, unknown>): AveragePoolAttributes => {
   const countIncludePad = (attributes.count_include_pad as number) === 0 ? false : true;
 
   const attr = parsePoolCommonAttributes(attributes);
   // TODO: support attribute 'ceil_mode'
   if (attr.ceilMode !== 0) {
     throw new Error('using ceil() in shape computation is not yet supported for AveragePool');
   }
-
-  return createAttributeWithCacheKey({countIncludePad, ...attr});
+  return {countIncludePad, ...attr};
 };
 
-export const averagePool = (context: ComputeContext, attributes: AveragePoolAttributes): void => {
+export const averagePool = (context: ComputeContext, attributes: Record<string, unknown>): void => {
```

Comment:
Can we still use `AveragePoolAttributes` since you have removed `AttributeWithCacheKey` in L272?

Reply:
This is possible when add below:
```
export interface FormatAttributes extends Record<string, unknown> {
```
I am not sure if this is an better option here.